authors/apps/articles/serializers.py

```diff
@@ -27,7 +28,45 @@ class ArticleSerializer(serializers.ModelSerializer):
         }
     )
 
+    def retrieve_author(self, obj):
+        """retrieve the author profile"""
+        pass
```

Comment:
@asirvex To improve your code, you can remove this method since you are not using it

Reply:
Thanks for the feedback, I'll remove it